src/test/javascript/spec/component/shared/user-settings/notification-settings.component.spec.ts

```diff
@@ -27,14 +30,17 @@ describe('NotificationSettingsComponent', () => {
         changed: false,
     };
 
-    const imports = [ArtemisTestModule];
+    const imports: any[] = [];
```

Comment:
is it possible to just remove that here? (Can you also leave it out in the ` TestBed.configureTestingModule({....`